.travis.yml

```diff
@@ -46,26 +53,33 @@ before_install:
     conda update conda
 
 install:
-  - conda env create --file environment.yml --name BiG-CZ
-  - source activate BiG-CZ
+  # We need to envs b/c the BiG-CZ software stack is not compatible with Pyhton 3 yet.
+  - conda env create --file webpage/environment.yml --name PAGE
+  - conda env create --file environment.yml --name BiG-CZ ;
```

Comment:
We need two envs b/c the publishing software is Python 3 only while most of the BiG-CZ stack is still Python 2.7 only.